jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -138,24 +138,30 @@ public Server(@Name("threadpool") ThreadPool pool)
     }
 
     /**
-     * @return True if {@link NetworkConnector}s are opened with {@link NetworkConnector#open()} during 
-     * start before handlers are started.
+     * @return True if the start sequence is verified by checking port availability prior
+     * to starting contexts and checking contexts prior to starting connectors.
```

Comment:
This description is awkward to me.
The prior version was more clear.

Are we binding or not?
What about windows and overlapping bind? (where windows allows you to bind to a port already bound by something else without error?)   _/me shakes fist at IIS_

Reply:
It is now doing a bit more, so I put a longer description on the setter, which I hope is clear?